### DISCUSSION TITLE
Is it possible to open two concurrent remote-server connections to IG Trading?

### DISCUSSION THREAD

**Discussion by kiann00** [ig-python/ig-markets-api-python-library#287] [Q&A]
Hi there, experts, ... a shot-in-the-dark.

Basically, I have two strategies I want to try out (in IG paper-trading). As I don't want to mix the 'running P&L', I try to run them in different demo accounts.

And I set each strategy to run via a python script in a remote-server, with each being a different account (one spreadbet, the other cfd). What I am noticing is that, I seem to be only able to connect to IG (for my remote server) *** ONE *** connection for each ip-address.

It doesn't seem to matter if they are either different accounts, or even a different login.

Does anyone have the same problem? If yes, is there any way to login to two different IG accounts from one remote-server/ip-address?

Basically, the endpoint is to be able to run two different strategies on two different accounts, using one remote-servier/ip-address....

🙏

**Comment by galburn** [comment]
I connect to two accounts (demo and live) simultaneously without any problem.  Perhaps you are somehow sharing the connection authentication details?  Can you share how you authenticate?
